cmd/generator/main.go

```diff
@@ -96,6 +96,7 @@ var generatorCmd = &cobra.Command{
 			"mattermost-plugin-gitlab",
 			"mattermost-plugin-nps",
 			"mattermost-plugin-webex",
+			"mattermost-plugin-todo",
```

Comment:
The list is actually deprecated and should not be used any longer. It just have caused to much conflicts when multiple plugins get released at the same time. I will file a ticket to remove it.

Reply:
Should I just remove that line? If I were to add other plugin without adding a line there, how would I do it? Just running `make plugin.json` did not add todo.

Reply:
Yes, that line should be removed.

You should run `go run ./cmd/generator/ add mattermost-plugin-todo v0.3.0 --beta` to add it.

I've filled https://github.com/mattermost/mattermost-marketplace/issues/74 to document this process.